firebase.json

```diff
@@ -1,10 +1,19 @@
 {
-  "functions": {
-    "predeploy": [
-      "npm --prefix \"$RESOURCE_DIR\" run lint"
-    ],
-    "source": "functions"
-  },
+  "functions": [
+    {
+      "predeploy": [
+        "npm --prefix \"$RESOURCE_DIR\" run lint"
+      ],
+      "source": "functions",
```

Comment:
can you undo all the changes in this file, because it was working fine with the old contents

Reply:
@shivareddy6  Reverted the changes of `firebase.json`